data/json/items/armor/storage.json

```diff
@@ -1258,7 +1347,89 @@
     "warmth": 10,
     "material_thickness": 2,
     "flags": [ "BELTED", "WATER_FRIENDLY" ],
-    "armor": [ { "encumbrance": [ 5, 20 ], "coverage": 70, "covers": [ "torso" ], "specifically_covers": [ "torso_hanging_back" ] } ]
+	"use_action": [ { "type": "attach_molle", "size": 8 }, { "type": "detach_molle" } ],
+    "armor": [ { "encumbrance": [ 6, 28 ], "coverage": 80, "covers": [ "torso" ], "specifically_covers": [ "torso_hanging_back" ] } ]
+  },
+  {
+    "id": "molle_large_rucksack",
+    "repairs_like": "bigback",
+    "type": "ARMOR",
+	"//": "Based upon own service and wikipedia: https://en.wikipedia.org/wiki/MOLLE",
+    "name": { "str": "MOLLE large rucksack" },
+    "description": "The Modular Lightweight Load-carrying Equipment is an advanced military large rucksack.  Covered with straps for modularity, it strikes a fine balance between storage space and encumbrance.",
```

Comment:
Isn't this the what the current "military rucksack" is supposed to be? I'd just apply the stats from this version onto that one instead of adding a new item.

Reply:
Perhaps, but there are regular rucksacks that don't have the molle straps as well. I figured it'd be so-so to leave them in since it will add a sort of variety to the kinds of rucks you can get.